toolchains/lowrisc_toolchain_rv32imc/clang_wrappers/nix/as

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
It appears several of these helper scripts are not referenced in this subdir's BUILD file or in the `tool_paths`.

Namely: as, c, c_ld, and cpp_ld.

Reply:
Yeah, I was trying to reconcile the toolchains/clang wrappers with the riscv32 toolchains, so I haven't figured out if I need these wrappers yet. My hope was I'd be able to see an unfulfilled requirement for them and add them or clean them up as necessary.